.github/workflows/release.yml

```diff
@@ -0,0 +1,119 @@
+name: Publish Python 🐍 distribution 📦 to PyPI and TestPyPI
+
+on: push
```

Comment:
Are you sure this should be run every commit?

Reply:
Yes.  We want to build each push but the process will stop at line 35 if this is not a release.

Reply:
Not sure why we need to build each push.

Reply:
You can try proposing changes to https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

Reply:
It says that it's "One popular choice". IMO build breaking while the normal CI doesn't break would be so rare that testing it only on tag push is enough (we don't need to release when we push a tag).

That said, I don't care about this facet too much.